x-pack/plugins/data_enhanced/server/search/es_search_strategy.ts

```diff
@@ -81,7 +81,7 @@ async function asyncSearch(
   const path = encodeURI(request.id ? `/_async_search/${request.id}` : `/${index}/_async_search`);
 
   // Wait up to 1s for the response to return
-  const query = toSnakeCase({ waitForCompletionTimeout: '100ms', ...queryParams });
+  const query = toSnakeCase({ waitForCompletionTimeout: '100ms', keepAlive: '1m', ...queryParams });
```

Comment:
Unrelated but it would be good to set the `batched_reduce_size` to a value greater than the default (`5`). I am saying this because we don't use partial results in Kibana at the moment so we don't need to create a partial results every 5 shards. Something like 32 or 64 should make the query faster and we can restore the default value when partial results are handled in Kibana ? It's also outside of the scope of this PR so I can open a new one if you like.

Reply:
Are there any risks associated with increasing this value?

Reply:
Nope, in blocking search the value is set to `512` so I think it's the other way around, `5` is too low and can have a negative impact on queries in terms of memory and speed. The only advantage to set a low value is for partial results so  we should set it higher until we expose partial results in Kibana.